gatsby-config.js

```diff
@@ -99,6 +99,7 @@ module.exports = {
           start_time: csvDatetimeParser,
           end_time: csvDatetimeParser,
           hide: csvBooleanParser,
+          contest: `number`,
```

Comment:
Would this replace contestid? Or do we need both for some reason?

Reply:
No. Yes. It's a little tragic.

`contestid` is the name of the field in `contests.csv`
`contest` is the name of the field in `findings.csv`

This was the apparent way to do parsing on a column. We're just kinda lucky that we don't have enough columns out there to collide.. yet?

We could 'simply' change the column name in `findings.csv` to be `columnid`, and this bit'd be unnecessary.

Reply:
Ah, I see. Ok, makes sense.